stock_manager/utils/constants.py

```diff
@@ -0,0 +1,22 @@
+from typing import Callable
+
+TOTAL_EQUATION: Callable[[int, int], int] = \
```

Comment:
If you're reusing a function, you should just define it.  Leaving it as a lambda prevents tracebacks from providing useful information, and it's not actually that much more typing at this point.